projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -47,6 +49,20 @@ const initAllForms = () => {
 		.forEach( initForm );
 };
 
+const IgnoredFieldsForEmptyForm = [
```

Comment:
Could there be more fields like that to ignore?

Reply:
In future we'll have "hidden field" input added in editor, which we need to include here some way. But that's future problems :-)

Another way to exclude would be "visibily hidden in browser" fields, either by `type="hidden"` or hidden via CSS. That might include all the weird stuff that people can do.

Reply:
To be clear, not super worried about weird stuff people do, this method with `.entries()` seems fine too.

Reply:
Of course. Removing the hidden fields is the way to go!